CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 1.75.0
+
+- Add `renamePropertyOn`, `unsetProperty`
+- Add `updateOwn`, `updateOwnOn`, `updateSome`, `updateSomeOn`, `updateAll`, `updateAllOn`
+- Add `crunchWhitespace`
+- Export `writeTreeNode`
+- Add `chunkByValue`
+- Pass indexes to `arrayToObject`
+- Add `omitByIndexed` conversion
```

Comment:
ADD:

`equalsAt`
`omitByPrefixes`
`isSubset`
`sizeBy`
`matchesBy`
`matchesBySome`
`mapValuesIndexed`

Reply:
Skipping `equalsAt` and `omitByPrefixes` due to low known usage